Tasks/AzureKeyVaultV1/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -42,5 +42,6 @@
   "loc.messages.CouldNotMaskSecret": "%s value has regular expressions hence could not mask completely",
   "loc.messages.CouldNotFetchAccessTokenforAzureStatusCode": "Could not fetch access token for Azure. Status code: %s, status message: %s",
   "loc.messages.CouldNotFetchAccessTokenforMSIDueToMSINotConfiguredProperlyStatusCode": "Could not fetch access token for Managed Service Principal. Please configure Managed Service Identity (MSI) for virtual machine 'https://aka.ms/azure-msi-docs'. Status code: %s, status message: %s",
-  "loc.messages.CouldNotFetchAccessTokenforMSIStatusCode": "Could not fetch access token for Managed Service Principal. Status code: %s, status message: %s"
+  "loc.messages.CouldNotFetchAccessTokenforMSIStatusCode": "Could not fetch access token for Managed Service Principal. Status code: %s, status message: %s",
```

Comment:
Why is this changed?

Reply:
I checked in any out-of-sync resource files that the build produced.  I can remove them if you'd rather, but since we usually commit them I just added them.